src/routes/conversation/[id]/+page.svelte

```diff
@@ -43,6 +44,7 @@
 			{
 				id: messageId,
 				is_retry: isRetry,
+				useSearch: $webSearchParameters.useSearch,
```

Comment:
Let's keep casing consistent among the parameters.

Here it's snake case because the other options passed to inference API are snake case too

Reply:
Fixed!